src/test/java/io/weaviate/client/WeaviateAuthClientTest.java

```diff
@@ -115,7 +115,7 @@ public void test503Case() {
       .verify(
         request()
           .withPath(OIDC_PATH),
-        VerificationTimes.exactly(3)
+        VerificationTimes.exactly(6)
```

Comment:
what is the motivation for change here?

Reply:
for some reason now we are sending 2x more of the requests... I was't able to find the reason behind this